packages/salesforcedx-apex-debugger/test/unit/core/sessionService.test.ts

```diff
@@ -142,7 +142,7 @@ describe('Debugger session service', () => {
           '{"message":"There was an error", "action":"Try again"}'
         );
         expect(service.isConnected()).to.equal(false);
-        expect(service.getSessionId()).to.an('undefined');
+        expect(service.getSessionId()).to.equal('');
```

Comment:
@DatGuyJonathan - Could you do a quick sanity check here? Reason for asking is that the unit tests explicitly check for undefined and now we change it to be `''`.

Reply:
These look appropriate.